website/versioned_docs/version-0.1.0/Connection/5.mongodb_connection.md

```diff
@@ -0,0 +1,137 @@
+

```

Comment:
- remove first empty line

Please run the documentation and look there that it looks ok